wiki/osu!stream/de.md

```diff
@@ -0,0 +1,375 @@
+- Ein Miss wird vergeben, wenn ein Hit-Circle zu früh oder überhaupt nicht getroffen wurde.
+  - Wenn das Objekt sehr, sehr früh getroffen wird, wackelt es stattdessen. Dabei passiert technisch gesehen nichts, außer dass man visuell erkennen kann, dass es zu früh angetippt wurde.
+
+**Hold-Circle**
+
+- Die Bewertung des Hold-Circles hängt von der anfänglichen Berührung und der Haltezeit vor der Explosion ab.
+  - 300 bei perfekter Ausführung.
+  - 100 für einen weniger präzisen anfänglichen Treffer und das richtige Halten.
+  - 50 dafür, dass er zumindest berührt wurde.
+  - 30 für jede Explosion.
+  - Der Multiplikator wird zurückgesetzt, wenn die Kreise nicht korrekt gehalten wurden. Ein Miss wird vergeben, wenn die Kreise überhaupt nicht gehalten wurden.
+
+**Slider**
+
+- **Die Bewertung von Slidern hängt von der Genauigkeit der anfänglichen Berührung ab**.
+- Slider bestehen aus Sliderticks, die den Start, das Ende und die repeat points des Sliders einbeziehen.
+  - Einen 300er, wenn man alle Sliderticks erreicht und einen genauen ersten Treffer hat.
+  - Einen 100er, wenn mindestens die Hälfte des Sliders mit einem guten ersten Treffer getroffen wurde.
+  - Einen 50er, wenn mindestens ein Slidertick oder der erste Treffer getroffen wurde.
+- **Einen Slider viel zu früh zu tippen** wird keinen Miss erzeugen, aber **wird den Punktemultiplikator auf 0 zurücksetzen**.
+- Ein Miss, wenn der Slider überhaupt nicht gehalten wurde.
+
+**Spinner**
+
+- Für einen Spinner werden 300er, 100er oder 50er vergeben, abhängig von der Anzahl der gemachten Spins verglichen mit der Länge des Spinners.
+- Ein Miss, wenn die Spinnermessleiste, die auf beiden Seiten des Spinners zu sehen ist, nicht gefüllt wurde.
+
+### Multiplikator
+
+Das Folgende erhöht den Punktemultiplikator um jeweils einen Punkt:
+
+- Treffer auf einem Hit-Circle.
+- Der Start, das Ende und jeder Hold-Tick eines Hold-Circles.
+- Der Start, das Ende und jeder Slidertick eines Sliders.
+- Einen Spinner vervollständigen.
+
+Das Folgende setzt den Punktemultiplikator auf null zurück:
+
+- Einen Hit-Circle verfehlen.
+- Den Hold-Circle loszulassen, bevor dieser explodiert ist.
+- Die verbundenen Beats nicht gleichzeitig treffen. (Einen treffen und den anderen verfehlen)
+- Einen Slider zu früh klicken.
+- Den Sliderstart oder einen Slidertick verfehlen.
+- Die Spinnermessleiste nicht vollständig füllen.
+
+Das Folgende wird den Punktemultiplikator weder erhöhen noch zurücksetzen:
+
+- Ein Sliderende verfehlen. (Es wird ein Good! oder ein 100/50 Treffer vergeben)
+- Punktebonus eines Spinners
+
+### Benotung
+
+![](img/Os-Grade.jpg "Ergebnisbildschirm")
+
+**Normale Noten**
+
+- SS = 100% Genauigkeit
+- S = Über 90% Genauigkeit
+- A = Über 80% Genauigkeit
+- B = Über 70% Genauigkeit
+- C = Über 60% Genauigkeit
+- D = Alles andere.
+
+### Lebensanzeige
+
+Nicht wie in osu!, ist die Lebensanzeige anders in osu!stream, da sie auf dem Spielmodus basiert.
```

Comment:
Honestly, the original sentence already is formulated in a redundant way imo. I would propose either one of the following:

```suggestion
Im Gegensatz zu osu! basiert die Lebensanzeige in osu!stream auf dem Spielmodus.
```

```suggestion
Die Lebensanzeige in osu!stream ist anders, da sie im Gegensatz zu osu! auf dem Spielmodus basiert.
```
(Closer to the original)